src/main/java/com/ProjectEureka/backend/models/Question.java

```diff
@@ -0,0 +1,53 @@
+package com.ProjectEureka.backend.models;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Document(collection = "Questions")
+public class Question {
+    @Id
+    private String id;
+
+    private String title;
+
+    @Field("question_date")
+    private String questionDate;
+
+    private String description;
+
+    private String category;
+
+    @Field("media_urls")
+    private List<String> mediaUrls;
+
+    @Field("status")
+    private String status;
+
+    private Boolean visible;
+
+    @Field("user_id")
+    private Long userId;
```

Comment:
I believe user_id must a string, according to the user table. Although Answer table has it incorrect too, just change it in your model only

Reply:
All in all works fine, good job!

Reply:
Done, good sir!